src/adapters/rpc-server/services/get-pending-friendship-requests.ts

```diff
@@ -1,13 +1,18 @@
-import { Empty } from '@dcl/protocol/out-ts/google/protobuf/empty.gen'
 import { RpcServerContext, RPCServiceContext } from '../../../types'
-import { FriendshipRequestsResponse } from '@dcl/protocol/out-ts/decentraland/social_service_v2/social_service.gen'
+import {
+  PaginatedFriendshipRequestsResponse,
+  GetFriendshipRequestsPayload
+} from '@dcl/protocol/out-ts/decentraland/social_service/v3/social_service_v3.gen'
 
 export function getPendingFriendshipRequestsService({ components: { logs, db } }: RPCServiceContext<'logs' | 'db'>) {
   const logger = logs.getLogger('get-pending-friendship-requests-service')
 
-  return async function (_request: Empty, context: RpcServerContext): Promise<FriendshipRequestsResponse> {
+  return async function (
+    request: GetFriendshipRequestsPayload,
+    context: RpcServerContext
+  ): Promise<PaginatedFriendshipRequestsResponse> {
```

Comment:
NIT: I'm thinking that maybe we could wrap-up pagination functionality in something like:

`Paginated<FriendshipRequestResponse>`

And so on for all the things we do need to paginate, what do you think?

Reply:
In terms of the functionality, I agree, but the types come from the protocol, which doesn't allowtheo create generics